marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/saveSavedQuery.sjs

```diff
@@ -18,11 +18,18 @@
 declareUpdate();
 
 const ds = require("/data-hub/5/data-services/ds-utils.sjs");
+const Security = require("/data-hub/5/impl/security.sjs");
 
 var saveQuery;
 var userCollections = ["http://marklogic.com/data-hub/saved-query"];
 var queryDocument = JSON.parse(saveQuery);
 
+let authorized = new Security().getRolesAndAuthorities().authorities.includes("canManageSavedQuery");
```

Comment:
Pinging @ryanjdew - this is doing what xdmp.securityAssert does, but that needs a privilege. I'll get a PR together that has an example of this.